### ISSUE TITLE
Content review from Check in

### ISSUE THREAD

**Issue by benbrasso-agile6** [department-of-veterans-affairs/va.gov-team#80519]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Check in
- **OCTO product owner:** Kay Lawyer
- **Product name:** Check in
- **Product manager:** Lori Pusey
- **Slack channel:** check-in-experience
- **Dedicated content writer on your team (if you have one):** shared
- **Dedicated a11y specialist on your team (if you have one):** shared
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

How might eCheck-in change when a BT eligibility API exists? 

[Userflow](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1711130372680/af42986654957f5cb47bf421ff89909c5595ecae?sender=u37bb983bd3fc3cc00c7d3286) 

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

Content updates to two existing pages in our web app. 1) Would you like to file for BT page, and 2) Check in complete page. 

This is future work; release date end of 2024. 

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.

Note: Attending design office hours to understand if using the [check eligibility pattern](https://design.va.gov/patterns/help-users-to/check-eligibility) is being used correctly. 

- [Variations of "Would you like to file for BT" page](https://www.figma.com/file/XYoHvDxgB2BpZIo2rrhFba/Check-in-%7C-Check-in?type=design&node-id=3388-4819&mode=design&t=HZUujAc5NhIc7aNf-4)
- [Variations of "Check in complete" page](https://www.figma.com/file/XYoHvDxgB2BpZIo2rrhFba/Check-in-%7C-Check-in?type=design&node-id=3389-6275&mode=design&t=HZUujAc5NhIc7aNf-4)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [x] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by benbrasso-agile6** [comment]
I would prioritize #80754, which should ideally go out with our product release. That one is more "writing" vs. "reviewing" since that page is owned by CAIA (I think), but we're happy to collab and do a first draft if that's better. 

Then, #80519 is very much not a priority, but that is reviewing 2 pages; 1 page with 3 alert-content variations, the other page with 2 body-content variations.

**Comment by benbrasso-agile6** [comment]
@loripusey I just dropped the HCE label so it doesn't appear in our EOS reports

**Comment by loripusey** [comment]
ok, I guess, those reports are causing us to change how we do things not in a good way

**Comment by laurwill** [comment]
Hey @benbrasso-agile6 and @loripusey , I think this work is long ago complete and we can close this issue, is that right?

**Comment by benbrasso-agile6** [comment]
Yeah; we worked on this async together. Can close.
